CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 3.14.0 (2020-03-30)
+
+- Added the `enablePointerEvents` option to `SkyOverlayConfig` which enables click events to pass through the overlay's backdrop. [#148](https://github.com/blackbaud/skyux-core/pull/148)
```

Comment:
Seems like `which enables` should be `to enable`. I mean, it's the option that enables this, not the configuration, right?